### ISSUE TITLE
[New build - DEFAULT] 22w01a: Various Features

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Starship-EVO#4437]
Hotfixes build with a few new features. Happy new year to all the players!

New Features:
 - Missions: It is now possible to view the list of current missions into the new Mission Tab from the Codex.
 - Missions: Untracking and tracking will hide and restore all hud information.
 - Tutorial: It is now also possible to cancel the starting tutorial.
 - Tutorial: New welcome message with a few instructions.
 - NPCs: A basic collision avoidance behaviour was added.
 - Player Customization: a preview of the player was added.
 - Player Customization: Alien race now have a proper name. You can suggest better names as those are still wip!

Hotfixes:
#4408 Some blueprint might fails to load.
#4406 Large link network can make the game freeze.
#4385 Prefab thumbnail not showing.
#4405 Docking on children entity was not working correctly.
#4401 Player can get stuck around Space Habitat.
#4398 Out of bounds widget incorrect computation for larger resolution.
#4383 Prefab event gate and turret are not initialized correctly after a paste.
#4382 #4381 Copy / pasting addon incorrect initialization.
#4374 World BVH debug anomaly.
#4370 Text Screen on Weapons incorrect parenting.
#4417 Shipyard welcome message has wrong name.
#4421 Shipyard event does not deactivate.

**Comment by TIKIRobo** [comment]
https://user-images.githubusercontent.com/56371294/147894087-9de5d5d2-bcc5-4b8f-b1b6-ac41b9fdf33c.mp4

today i learned there is flower/ear/hair physics in the game!

**Comment by pinesh** [comment]
wooo!

**Comment by ZachZent** [comment]
In the distant future for max customization options, I'd suggest a mix of FTL and Stellaris for model customization.

Have different "categories" of races such as Mammalian, Aquatic, etc. With more alien models that are added, they can go in these categories so that you have multiple mammalian races, multiple avian races. That way, if you so choose, different kingdoms can have FTL like stat bonuses/detriments like run speed. Makes things a bit more dynamic.

Also for an extra layer of variety, I'd have primary and secondary color sliders. Blue skin human with red hair. Purple chloroplast platnoid with pink petals. Really will make the universe come a bit more alive when you have that sort of variety

**Comment by tsunamayo** [comment]
yep for sure it needs some color, that would be the first low hanging fruit regarding customization...

**Comment by thatisjake** [comment]
thanks for fixing the bug! My gf misses me.. I've been playing this game too long...

**Comment by Briaireous** [comment]
> thanks for fixing the bug! My gf misses me.. I've been playing this game too long...

REJECT RELATIONSHIPS, RETURN TO SINGULARITY

**Comment by Kaiser-Indrasil** [comment]
Thanks for fixing the bugs, Tsuna!

**Comment by Horkas** [comment]
@tsunamayo So what kind of feedback and testing are you looking for specifically now? Is there anything in particular you would like to be tested? Any subjects you want feedback on?
